src/sentry/tasks/store.py

```diff
@@ -87,6 +87,17 @@ def submit_symbolicate(project, from_reprocessing, cache_key, event_id, start_ti
     task.delay(cache_key=cache_key, start_time=start_time, event_id=event_id)
 
 
+def submit_symbolicate_low_priority(
+    project, from_reprocessing, cache_key, event_id, start_time, data
+):
+    task = (
+        symbolicate_event_from_reprocessing_low_priority
+        # if from_reprocessing
+        # else symbolicate_event_low_priority
```

Comment:
i was thinking of ignoring reprocessing for the lowpriority and only use it for brand new events.  but... maybe that's naive and you're right.  It does mean we have to make sure to handle a bunch more cases.

Reply:
Yeah, my impression was that there's nothing that explicitly prevents events in the low priority queue from being reprocessed. It doesn't make sense to promote reprocessed LPQ events up to the regular queue for obvious reasons, and my understanding was that we deliberately have a separate code path for reprocessing for metrics or tracking purposes.

What are some of the cases you're thinking of as a result of this? I'll admit that I have a fairly surface-level understanding of reprocessing right now. I'm assuming this diff doesn't cover most of them.